test/Unit/Util/GitTest.php

```diff
@@ -72,7 +72,9 @@ public function testRemoteUrlsReturnsRemoteUrls(): void
         while (3 > \count($this->remoteUrls)) {
             do {
                 $remoteName = $faker->unique()->word;
-            } while (\in_array($remoteName, \array_merge($remoteNames, $this->remoteUrls), true));
```

Comment:
This doesn't even make sense, as the keys in `$this->remoteUrls` are the remote names, see line `77`.